public/images/direction_icons/corner-up-left-grey.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="#353c52" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-corner-up-left"><polyline points="9 14 4 9 9 4"></polyline><path d="M20 20v-7a4 4 0 0 0-4-4H4"></path></svg>
```

Comment:
color is not consistent with the geolocation button (`#5c6f84` vs `#353c52`)
Ideally we would add this to the icomoon font (that already contains "corner-up-right")

Reply:
I removed the colored svgs,
I fixed the colors,
I used the corner-up-right icon from the font (we can change it when the up-left icon will be added in the font)